### ISSUE TITLE
Authentication rejected

### ISSUE THREAD

**Issue by emeraldservices** [ultravnc/UltraVNC#127]
I am new here
I am trying to connect to Win Server 2022 firewall to allow the port allow and I am trying to connect and shows me 
error authentication rejected

**Comment by Neustradamus** [comment]
@emeraldservices: Have you added the VNC port in Windows firewall?

**Comment by Neustradamus** [comment]
@emeraldservices: What is your current situation?

**Comment by Neustradamus** [comment]
@emeraldservices: It is possible to have an answer?

**Comment by Neustradamus** [comment]
@emeraldservices: Can you try a recent UltraVNC version > 1.6?

**Comment by Neustradamus** [comment]
@emeraldservices: Can you try the latest UltraVNC version?
- https://uvnc.com/

**Comment by jygarneau** [comment]
Yes, I changed nothing on this tab.

**Comment by jygarneau** [comment]
The "%ProgramData%/UltraVNC " is not there". I created the directory and copied "%ProgramFiles%\uvnc_bvba\UltraVNC\ultravmc.ini". The service is now working!
Thank you for your support. I suggest to update the installer.

**Comment by RoboBurned** [comment]
Same issue with 1.6.4.0 with service installed. There was no file in ProgramData\UltraVNC. When I moved the ultravnc.ini file from ProgramFiles to ProgramData then I was able to connect.
The problem is that when I try to change settings, they are saved to the ProgramFiles, so I have to move the file again.

**Comment by Neustradamus** [comment]
@RoboBurned: Have you progressed on your problem?

**Comment by RoboBurned** [comment]
@Neustradamus Yes. I removed config from ProgramFiles and AppData and created in ProgramData.
Then it started to work fine.

**Comment by Neustradamus** [comment]
@RoboBurned: Thanks for your confirmation! :)
